### ISSUE TITLE
Report val loss aggregated by data origin

### ISSUE THREAD

**Issue by mrcabbage972** [ontocord/MDEL#33]
When training an expert model on a mix of domain and pile data, we would like to log the validation loss for the domain data and the pile data separately.
